components/settings/settings.cpp

```diff
@@ -354,12 +355,14 @@ float Manager::getFloat (const std::string& setting, const std::string& category
     const std::string value = getString(setting, category);
     try
     {
-        return std::stof(value);
+        // We have to rely on Boost because std::stof from C++11
+        // uses the current locale for separators which we don't want and often silently ignores parsing errors.
```

Comment:
This might not be the best place to break the line - because the first line is much shorter than the second, if someone isn't paying much attention, it could look like we're avoiding `std::stof` *because* it's from C++11 and that the second line of the comment is something unrelated.